doc/classes/OS.xml

```diff
@@ -289,10 +289,9 @@
 			</description>
 		</method>
 		<method name="get_granted_permissions" qualifiers="const">
-			<return type="PackedStringArray" />
+			<return type="int[]" />
 			<description>
 				On Android devices: Returns the list of dangerous permissions that have been granted.
-				On macOS: Returns the list of user selected folders accessible to the application (sandboxed applications only). Use the native file dialog to request folder access permission.
```

Comment:
Returning accessible folders from a permission related method on macOS doesn't feel quite match up, so I removed it. Is such functionality still relevant?

Reply:
It is related to permissions, it is a list of folders access permissions were granted for. It's definitely should not be removed, since it's the only way to know if you still have permission to access the folder (or need to ask the user to grant it again).

Reply:
Having different behaviours on the same method feels quite strange...

Will moving it to other class like `DirAccess` help?